docs/comparison-analysis.md

```diff
@@ -58,6 +58,7 @@ The relevance test run summary is determined by the number of significant and re
 
 Magnitude is a combination of two factors:
 * how large a change is regardless of the direction of the change
+  * The point at which a change is considered in a different magnitude category depends on the metric being analyzed.
```

Comment:
I don't understand this sentence. Reword?

Reply:
Expanded and reworded. Let me know what you think.